submissions/ogennaOgbogu/CSS/signupStylesheet.css

```diff
@@ -1,456 +1,353 @@
-
 * {
     box-sizing: border-box;
 }
 
-html,body
-
-{
+html,body {
     width: 100%;
     height: 100%;
     margin: 0;
     padding: 0;
-    overflow-x: hidden;
+    overflow-x: auto;
+    overflow-y: auto;
```

Comment:
Instead of doing both `overflow-x: auto; overflow-y: auto;` just do `overflow: auto`

Reply:
Thank you @adaugochi, done.